src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1263,139 +957,185 @@
+      For a list of available lock Ids, run `az group lock list --resource-group MyResourceGroup -o json`.
     text: >
-        az group lock show -n lockname -g MyResourceGroup
+        az group lock show --ids /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Authorization/locks/MyLockName1 \\
+            /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Authorization/locks/MyLockName2 
 """
 
 helps['group lock update'] = """
 type: command
-short-summary: Update a resource group lock.
+short-summary: Update a resource group lock with new notes, or change the lock type.
 examples:
   - name: Update a resource group lock with new notes and type
     text: >
-        az group lock update --name lockName -g MyResourceGroup --notes newNotesHere --lock-type CanNotDelete
+        az group lock update --name MyLockName --resource-group MyResourceGroup --notes newNotesHere --lock-type CanNotDelete
+  - name: Update a resource group lock using the Id.  For a list of available lock Ids, run `az group lock list --resource-group MyResourceGroup -o json`.
+    text: >
+        az group lock update --ids /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Authorization/locks/MyLockName --notes MyNewNote
+  - name: Update several resource group locks at the same time using a space delimited list of Ids.
+    text: >
+        az group lock update --ids /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Authorization/locks/MyLockName1 \\
+            /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/MyResourceGroup/providers/Microsoft.Authorization/locks/MyLockName2 \\
+            --notes MyNewNoteForAllIDs
 """
 
 helps['group update'] = """
 type: command
 short-summary: Update a resource group.
 examples:
-  - name: Update a resource group. (autogenerated)
+  - name: Add tags to a resource group.
+    text: |
+        az group update --resource-group MyResourceGroup --set tags.Department1='{"Department name":"IT","Cost Center":"tech002"}'
+        az group update --resource-group MyResourceGroup --set tags.Environment=DEV
+  - name: Add tags to a resource group preserving string literals.
+    text: |
+        az group update --resource-group MyResourceGroup --set tags.Department1='{"Department name":"IT","Cost Center":"tech002"}' --force-string
+  - name: Remove tags from a resource group.
     text: |
-        az group update --resource-group MyResourceGroup --set tags.CostCenter='{"Dept":"IT","Environment":"Test"}'
-    crafted: true
+        az group update --resource-group MyResourceGroup --remove tags
+  - name: Add a property to a resource group.
+    text: |
+        az group update --resource-group MyResourceGroup --add properties.MyPropertyName=MyValue NEED WORKING EXAMPLE HERE 
```

Comment:
@zhoxing-ms Looking at [arm.py#L389-L391](https://github.com/Azure/azure-cli/blob/8779b435ba3d9f920986f7c7990f717b58507a51/src/azure-cli-core/azure/cli/core/commands/arm.py#L389-L391) ...

add_usage = '--add property.listProperty <key=value, string or JSON string>'
set_usage = '--set property1.property2=<value>'
remove_usage = '--remove property.list <indexToRemove> OR --remove propertyToRemove'

... I am intuitively trying to get `--set properties.MyNewPropertyName` to work like `--set tags.MyNewTagName`.  Can you please provide a working example for `--add` and `--set` as I am obviously missing the connection?

![image](https://user-images.githubusercontent.com/58762114/199599301-607c598d-44e9-4543-8d4f-a1a92232a78c.png)

Reply:
For `--set`, this is an example: `az group update -g zhoxing-test --set tags.key=value`
![image](https://user-images.githubusercontent.com/16612065/199641129-25687660-0466-48e6-a819-eb553330520c.png)

Reply:
For the `--add` parameter, it's usage scenario is to add elements to an array type property. Currently, the resource group does not have such property. This parameter is only added by `generic_update_command` framework, I believe customers have no need to use it, so it should not be necessary to add an example for it

Reply:
@zhoxing-ms, is the `--add` parameter then an example of where we should set `is-visible = false` somewhere so it doesn't even show in the autogenerated content?

I'll remove my examples, but I feel that parameters a customer won't (can't?) use just makes noise.  What is your perspective?

Reply:
Good suggestion! I think we can consider hiding the help message of this parameter `--add` through `c.deprecate(hide=True)`